workflow/rules/common.smk

```diff
@@ -608,6 +608,8 @@ def get_tabix_params(wildcards):
     if wildcards.format == "txt":
         return "-s 1 -b 2 -e 2"
     if wildcards.format == "tsv":
+        if config["ref"]["build"] == "GRCh37":
+            return "-f -s 1 -b 2 -e 2"
```

Comment:
Why the special case? At least this must be documented.

Reply:
When indexing the revel-score file the parameters are different as the file holds regions for hg38 and hg19.
Depending on which reference is being used parameters for indexing differ.